### DISCUSSION TITLE
Date of Project Synopsis and Project

### DISCUSSION THREAD

**Discussion by amrutmuj** [github/india#534] [GitHub Platform]
I've been looking since 30th but I'm unable to get access to Project Synopsis and Project dashboard.
Does anyone of you've got access?

I've completed my industry session and it is marked as complete but still

**how many of you got access to Project Synopsis and Project dashboard?**

![image](https://user-images.githubusercontent.com/85221126/139795157-78936c7d-328e-4745-b9d6-a99eedbd20a9.png)

**Comment by ghost** [comment]
Same, I haven't gotten it either :/

**Comment by FRTMentor** [comment]
Please reach out our support team.
support@futurereadytalent.in
